website/www/site/content/en/get-started/tour-of-beam.md

```diff
@@ -16,7 +16,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Tour of Beam
+# An Intro to Beam (Python)
 
 Here you can find a collection of the interactive notebooks available for Apache Beam, which are hosted in
 [Colab](https://colab.research.google.com).
```

Comment:
We need to update the links on this page to the colab notebooks (keyword search tour-of-beam on this page in your editor)